### PR TITLE
KO translation for visual_story/animating_elements.md.

### PR DESCRIPTION
**Pull request by captain-lee**
Screenshot of translated:
screenshot from 2018-08-14 08-59-23

### CODE REVIEW

**Comment by captain-lee** [comment]
@techhtml, please take a look.

File: content/docs/design/visual_story/animating_elements@ko.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+{{ anim('/static/img/docs/tutorials/amp_story/pg5-collage-animation.gif', 720, 1280, align='center third', alt='Page 5 collage', poster='/static/img/docs/tutorials/amp_story/pg5-collage.png' ) }}
+
+이제 여러개의 효과를 섞거나 잇는 등 다양한 방법으로 애니메이션을 만들수 있게
+되었습니다. 본 자습서는 기본적인 것만 다루어봤는데,
+[AMP 스토리 레퍼런스 문서](/ko/docs/reference/components/amp-story.html#animations)에
+더 자세한 내용이 있으니 참고하십시오.
+
+<div class="prev-next-buttons">
+  <a class="button prev-button" href="/ko/docs/design/visual_story/add_more_pages.html"><span class="arrow-prev">Prev</span></a>
```
Comment (bpaduch) [inline content/docs/design/visual_story/animating_elements@ko.md]: Translate "Prev"

File: content/docs/design/visual_story/animating_elements@ko.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+  <a class="button next-button" href="/ko/docs/design/visual_story/create_bookend.html"><span class="arrow-next">Next</span></a>
```
Comment (bpaduch) [inline content/docs/design/visual_story/animating_elements@ko.md]: Translate "Next"

File: content/docs/design/visual_story/animating_elements@ko.md (hunk 1 above)
Comment (captain-lee) [reply content/docs/design/visual_story/animating_elements@ko.md]: Fixed.

File: content/docs/design/visual_story/animating_elements@ko.md (hunk 2 above)
Comment (captain-lee) [reply content/docs/design/visual_story/animating_elements@ko.md]: Fixed.

File: content/docs/design/visual_story/animating_elements@ko.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+0.5초안에 애니메이션이 끝나는 코드입니다.
+
+```html
+<amp-story-page id="my-page">
+  ...
+  <p class="my-element"
+      animate-in="fly-in-left"
+      animate-in-delay="0.3s"
+      animate-in-duration="0.5s">
+    왼쪽에서부터 날아오는 문구
```
[the remark this answers is not in the document]
Comment (captain-lee) [reply content/docs/design/visual_story/animating_elements@ko.md]: Yep, reverted back to the original.

File: content/docs/design/visual_story/animating_elements@ko.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+
+```html
+<amp-story-page id="page3">
+  ...
+  <amp-story-grid-layer template="vertical">
+    <p animate-in="drop">Drop this text into the page</p>
+</amp-story-page>
+```
+
+{% call callout('For fun', type='success') %}
```
[the remark this answers is not in the document]
Comment (captain-lee) [reply content/docs/design/visual_story/animating_elements@ko.md]: Translated.

**Comment by captain-lee** [comment]
Thanks @bpaduch for the review!

**Pull request merged**